main.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/urfave/cli"
+)
+
+func main() {
+	var network, rpcUrl, function, contractAddress string
+	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stdout})
+	app := cli.NewApp()
+	app.Name = "web3-cli"
+	app.Version = "0.0.1"
+	app.Usage = "web3 cli tool"
+	globalFlags := []cli.Flag{
```

Comment:
What about:
```suggestion
	app.Flags = []cli.Flag{
```
and remove it from all the individual commands?
Then this works: `web3-cli -network gochain-testnet bl 0`